docs/workflow/Codespaces.md

```diff
@@ -7,7 +7,16 @@ dotnet/runtime runs a nightly GitHub Action to build the latest code in the repo
 
 **NOTE**: In order to use a prebuilt codespace, when you create your machine be sure to select an **`8 core`** machine.
 
```

Comment:
NIT: maybe remove this note as it is mentioned in step 2 
```suggestion
```

Reply:
I felt it was important enough to say it twice. Since this is not the default, and GH Codespaces currently doesn't give you an indication which machine types have a pre-built available, it needed repeating.